src/App.js

```diff
@@ -32,6 +33,7 @@ export default function App() {
             <HomePage
               savedPlants={savedPlants}
               onHandleBookmarkClick={handleBookmarkClick}
+              onDeleteCard={handleDeletePlant}
```

Comment:
Just to be consistent with your names 🙂 
```suggestion
              onDeletePlant={handleDeletePlant}
```